templates/user-research/index.html

```diff
@@ -0,0 +1,201 @@
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-nav">Join a navigation study</a></p>
+      </div>
+    </div>
+    <div class="row p-block">
+      <div class="col-3 col-medium-2"></div>
+      <div class="col-9 col-medium-4">
+        <hr class="p-rule">
+        <p class="p-heading--2">Ubuntu Security</p>
+        <p>We would love to learn how you manage your security and updates, and the way you use our CVE and security notices.</p>
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-security">Help us improve security</a></p>
+      </div>
+    </div>
+    <div class="row p-block">
+      <div class="col-3 col-medium-2"></div>
+      <div class="col-9 col-medium-4">
+        <hr class="p-rule">
+        <p class="p-heading--2">Ubuntu Pro</p>
+        <p>We are updating Ubuntu Pro with new features and coverage, to make it more powerful and intuitive for our customers.</p>
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-pro">Tell us how Ubuntu Pro works for you</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-muted-heading">How does it work?</h2>
+      </div>
+      <div class="col-9 col-medium-4 p-block">
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--2">Sign-up</p>
+          </div>
+          <div class="col-5">
+            <p>Fill in a <a href="">5-minute registration survey</a>, <br>to help us match you with the right studies.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-3 col-medium-2"></div>
+      <div class="col-9 col-medium-4 p-block">
+        <hr class="p-rule">
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--2">Receive invites</p>
+          </div>
+          <div class="col-5">
+            <p>We'll send you an invite when the appropriate study comes along. 
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-3 col-medium-2"></div>
+      <div class="col-9 col-medium-4 p-block">
+        <hr class="p-rule">
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--2">Take part</p>
+          </div>
+          <div class="col-5">
+            <p>You can decide if you want to (remotely) participate. There's usually a choice of days and times for interviews &ndash; you may also get invitations to surveys.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section is-deep">
+    <hr class="p-rule is-fixed-width">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <p class="p-muted-heading">Talk to us!</p>
```

Comment:
This heading isn't on the doc

Reply:
this one was added by Lyubo and has gone past user research so should be good. I will ask Martin to update the doc to reflect these changes